umbraco-cloud/set-up/project-settings/umbraco-cicd/samplecicdpipeline/azure-devops-pwsh.md

```diff
@@ -0,0 +1,147 @@
+---
+description: >-
+  This section provides a step-by-step guide to setting up a CI/CD pipeline in
+  Azure DevOps using the provided sample Powershell scripts.
+---
+
+# Azure DevOps with Powershell scripts
+
+Before setting up the pipeline in Azure DevOps, make sure that the steps in the [Configuring a CI/CD pipeline](./) are done.
+
+You will need to define your pipeline in YAML, and find a way to interact with the Umbraco Cloud API.
+
+
+{% hint style="info" %}
+We have created a sample pipeline for Azure DevOps.  It includes YAML-files and custom Powershell scripts to interact with the Umbraco Cloud API.
+
+You can get the samples for Azure DevOps and GitHub from this [repository](https://github.com/umbraco/Umbraco.Cloud.CICDFlow.Samples).
+
+For Azure DevOps you will need the following files:
+- From the root folder
+  - `cloud.zipignore`
+- From `powershell` folder
+  - `Get-LatestDeployment.ps1`
+  - `Get-ChangesById.ps1`
+  - `New-Deployment.ps1`
+  - `Add-DeploymentPackage.ps1`
+  - `Start-Deployment.ps1`
+  - `Test-DeploymentStatus.ps1`
+- From the `powershell/azuredevops` folder
+  - `azure-release-pipeline.yml`
+  - `cloud-sync.yml` 
+  - `cloud-deployment.yml`
+
+{% endhint %}
+
+## Import Cloud project repository to Azure DevOps
+Go to your repositories in Azure DevOps and click on "Create a repository".
+
+- Create a new empty repository (don't add a README and don't add a .gitignore), and note down the clone URL.
+- Go to the Umbraco Cloud Portal and clone your cloud project down locally. [This article](../../../working-locally.md#cloning-an-umbraco-cloud-project) describes how you can find the clone URL.
+- Now working locally remove the Git Remote called `origin`, which currently points to Umbraco Cloud
+
+ ```sh 
+ git remote remove origin
+ ```
+
+- Optionally rename branch `master` to `main`
+
+ ```sh 
+ # optional step
+ git branch -m  main
+ git symbolic-ref HEAD refs/heads/main
+ ```
+
+- Add a new remote called origin and pointing to the Azure DevOps clone URL and push
+
+ ```sh 
+ git remote add origin https://{your-organization}@dev.azure.com/{your-organization}/{azure-project-scope}/_git/{your-repository}
+ git push -u origin --all
+ ```
+
+Now we can move on to setting up a pipeline.
+
+## Set up the Azure DevOps pipeline files
+
+While working with the project on you local machine, do the following to prepare the pipeline.
+- Copy the `cloud.zipignore` file to the root of your repository
+- Make a copy of the `.gitignore` and call it `cloud.gitignore`
+  - Both files should be in the root of your repository
```

Comment:
note for Jesper so he remembers:
add a line about gitignore the git-patch.diff file non cloud .gitignore